Microsoft.Toolkit.Uwp.Services/Properties/Microsoft.Toolkit.Uwp.Services.rd.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
   <Library Name="Microsoft.Toolkit.Uwp.Services">
+    <Namespace Name="System.Text.Json.Serialization.Converters" Dynamic="Required All"/>
```

Comment:
The main Microsoft.Toolkit.Uwp package provides the https://docs.microsoft.com/en-us/windows/communitytoolkit/helpers/objectserializer that kinds of needs to be generic, and in order for this to be a less disruptive breaking change (requiring devs to manually add this to their rd.xml files, I've opted to just add them all to avoid the difference between this approach and Newtonsoft's).
I would be open to switch this back if the size of this was considerable, but I'm assuming it is only adding a few KBs (if that much), and its a fair tradeoff from the headache of knowing how to fix this and giving this burden back to developers.
What do you think?

Reply:
Yeah no, I 100% agree with that, the size difference should be pretty small anyway and I'd say making the transition effortless for consumers of this package is a pretty major point, even if this would be in a breaking change. Not everyone (if even anyone, at all) likes to have to go mess with the `.rd.xml` file in their apps 😄

Plus this only failing in Release with .NET Native could be another point of confusion for less experienced devs, so yeah I think just leaving this as is was definitely the right decision here 👍